### PR TITLE
Added Gonzo class

### PR DESCRIPTION
**Pull request by agonzo34**
Added a class named "Gonzzzz" and a test for it!

### CODE REVIEW

File: spec/business_spec.rb [hunk 1]
```diff
@@ -8,3 +8,12 @@
...
 end
+
+RSpec.describe Gonzo do
+  subject {Gonzo.new}
+  describe '#perform' do
+    it 'returns Gonzo' do
+      expect(subject.perform).to eq("gonzzzz")
+    end
+  end
+end
```
Comment (mbramson) [inline spec/business_spec.rb:19]: Looks great, where's the class?

**Review by mbramson — CHANGES_REQUESTED** [review]
Add the class under test

File: spec/business_spec.rb (hunk 1 above)
Comment (agonzo34) [reply spec/business_spec.rb:19]: Whoops!  
Errr, I think I correctly added it now?

File: business.rb [hunk 2]
```diff
@@ -2,4 +2,9 @@ class Business
   def perform
     "Business"
   end
+ 
+class Gonzo
+  def perform
+    "gonzzzz"
+  end
```
Comment (mbramson) [inline business.rb:9]: Solid!

**Review by mbramson — COMMENTED** [review]
Perfect!

**Pull request closed without merging**